src/main/java/org/tron/common/runtime/vm/program/Stack.java

```diff
@@ -59,4 +60,25 @@ public void swap(int from, int to) {
   private boolean isAccessible(int from) {
     return from >= 0 && from < size();
   }
+
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o.getClass() != this.getClass()) {
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: A "NullPointerException" could be thrown; "o" is nullable here. ([squid:S2259](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2259))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1681&issues=AWboX3TKLpqqSUlc8eNB&open=AWboX3TKLpqqSUlc8eNB)

[ISSUE_KEY]: AWboX3TKLpqqSUlc8eNB